src/create_cljs_app/template.cljs

```diff
@@ -5,12 +5,12 @@
              [copyFileSync mkdirSync readdirSync readFileSync statSync writeFileSync]]))
 
 (def template-dir
-  "Path there template files are."
+  "Path where template files are."
   (join js/__dirname ".." "template"))
 
 (def template-ignores
   "UNIX-style paths from the template dir to ignore. Mostly leftovers from template development."
-  #{".shadow-cljs" "node_modules" "out" "yarn.lock" "public/js"})
+  #{})
```

Comment:
Heya, I left a comment about this in https://github.com/filipesilva/create-cljs-app/issues/138#issuecomment-650771820.

I don't understand why this would stop ignoring the `.gitignore` file since it wasn't part of this list in the first place. It would stop ignoring a lot of other stuff though.